rabbitmq/resource_binding_test.go

```diff
@@ -136,6 +153,54 @@ resource "rabbitmq_binding" "test" {
     routing_key = "#"
 }`
 
+const testAccBindingConfig_slashesAreOkay = `
+resource "rabbitmq_vhost" "test" {
+    name = "/virtual//host///"
+}
+
+resource "rabbitmq_permissions" "guest" {
+    user = "guest"
+    vhost = "${rabbitmq_vhost.test.name}"
+    permissions {
+        configure = ".*"
+        write = ".*"
+        read = ".*"
+    }
+}
+
+resource "rabbitmq_exchange" "test" {
+    name = "test"
+    vhost = "${rabbitmq_vhost.test.name}"
```

Comment:
This needs to be

```hcl
vhost = "${rabbitmq_permissions.guest.vhost}"
```

This creates an implicit dependency on the permissions resource and so the exchange is created _after_ the permissions are set. Otherwise Terraform will create the exchange before the permissions are set and the test won't pass.

Reply:
Ohh, _nice_ catch! I'm pretty sure I actually ran into this problem while testing and didn't realize it.